dashboard.py

```diff
@@ -486,8 +486,23 @@ def create_account_clicked(first_name, last_name, new_email, new_password) -> No
 
 def track_clicked(user_id, url, notification_price) -> None:
     """Tracks product"""
+    error_present = False
     if not url or not notification_price:
         st.error("All fields are required to track a product.")
+        error_present = True
```

Comment:
Great error handling, will stop the page from crashing if users don't use it correctly.